### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by donovanclarke**
Chapter 40

### CODE REVIEW

File: app/assets/javascripts/alert.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
+  debugger
+  $(".alert").delay(800).fadeOut(800);
```
Comment (eomuraliev) [inline app/assets/javascripts/alert.js:3]: In the future, you might want to make these notices last more than .8 seconds.

File: app/assets/javascripts/alert.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+});
+
+
+// setTimeout(function(){
```
Comment (eomuraliev) [inline app/assets/javascripts/alert.js:7]: Also, avoid committing commented out code unless it is documentation

**Comment by eomuraliev** [comment]
Good job.

**Comment by donovanclarke** [comment]
Yes!, I think I realized I still had the debugger in the javascript / took out the commented out code a few hours later.

**Pull request merged**